draco/core/containers.py

```diff
@@ -272,6 +282,41 @@ def add_dataset(self, name):
 
         return dset
 
+    def chunkify(self):
```

Comment:
This should probably have a more sensible name: `ensured_chunked` would be more informative.